### DISCUSSION TITLE
Alembic upgrade head in infinite loop

### DISCUSSION THREAD

**Discussion by psxpa3** [miguelgrinberg/Flask-Migrate#487] [Q&A]
Hello,
I am trying to upgrade a table in my db by just one column. Even the database upgrade took just ~5-7 minutes to upgrade. However, this one column upgrade is taking foreever. I do not see any error at all. Could anyone please help me out what can be done?

**Comment by miguelgrinberg** [comment]
Please do not write issues on this package when you are looking for help with your own application. I'm moving this to a discussion, which is the right place for this.
